deps/github.com/openshift/kubernetes/openshift-hack/rebase.sh

```
@@ -125,51 +124,64 @@ fi
 
 # openshift-hack/images/hyperkube/Dockerfile.rhel still has FROM pointing to old tag
 # we need to remove the prefix "v" from the $k8s_tag to stay compatible
-sed -i -E "s/(io.openshift.build.versions=\"kubernetes=)(1.[1-9]+.[1-9]+)/\1${k8s_tag:1}/" openshift-hack/images/hyperkube/Dockerfile.rhel
+podman run --rm -v "$(pwd):/workspace:Z" docker.io/library/alpine:latest \
+  sed -i -E "s/(io.openshift.build.versions=\"kubernetes=)(1.[1-9]+.[1-9]+)/\1${k8s_tag:1}/" \
+  /workspace/openshift-hack/images/hyperkube/Dockerfile.rhel
 go_mod_go_ver=$(grep -E 'go 1\.[1-9][0-9]?' go.mod | sed -E 's/go (1\.[1-9][0-9]?)/\1/' | cut -d '.' -f 1,2) # Need to handle mod versions like 1.23 and 1.23.4; our release images only have major.minor
```

Shellcheck (0.11.0)

[warning] go_mod_go_ver appears unused. Verify use (or export if used externally).

(SC2034)